src/index.ts

```diff
@@ -247,7 +259,8 @@ export function Dictionary<V>() {
  */
 export function Record<O>(runtypes: {[K in keyof O]: Runtype<O[K]> }): Runtype<O> {
   return runtype(x => {
-    Dictionary().check(x)
+    if (x === null || x === undefined)
+      throw new ValidationError(`Expected a defined non-null value but was ${typeof x}`)
```

Comment:
Oh right, of course. In that case, let's use

```ts
Record({}).check(xs)
```

in the definition of `Dictionary`.